content/pages/sphinx.html

```diff
@@ -0,0 +1,243 @@
+        </div>
+      </div>
+    </a>
+
+    <a href="https://docs.jupyter.org/en/latest/" class="card">
+    <div class="ui medium centered image">
+        <img src="https://cdn.icon-icons.com/icons2/2699/PNG/512/jupyter_logo_icon_169453.png" alt="A random image">
+      </div>
+      <div class="content">
+        <div class="header">Jupyter</div>
+        <div class="meta">
+          Case study
+        </div>
+        <div class="description">
+            Jupyter Notebook is a web-based interactive computational environment for creating notebook documents.
+        </div>
+      </div>
+    </a>
+
+    <a href="https://docs.ceph.com/" class="card">
+    <div class="ui small centered image">
+        <img src="https://ceph.io/assets/bitmaps/Ceph_Logo_Standard_RGB_Black_120411_fa.png" alt="A random image">
+      </div>
+      <div class="content">
+        <div class="header">Ceph</div>
+        <div class="meta">
+          Case study
+        </div>
+        <div class="description">
+            Ceph uniquely delivers object, block, and file storage in one unified system.
+        </div>
+      </div>
+    </a>
+</div>
+
+<div class="ui very padded basic vertical segment">
+  <div class="ui horizontal divider">Proven across time</div>
+</div>
+
+{# Grid view stuff #}
+<div class="ui stackable equal width grid">
+  <div class="stretched row">
+    <div class="column">
+      <div class="ui very padded raised large segment">
+        <h4 class="ui header large">
+          1 million served. Every day.
+        </h4>
+
+        {#
+
+        Statistics are nice for important numbers
+        https://fomantic-ui.com/views/statistic.html
+
+        #}
+        <div class="ui two small statistics">
+          <div class="statistic">
+            <div class="value">
+              100K
+            </div>
+            <div class="label">
+              Projects
+            </div>
+          </div>
+          <div class="statistic">
+            <div class="value">
+              750M
+            </div>
+            <div class="label">
+              Yearly pageviews
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="column">
+      <div class="ui very padded inverted violet large segment">
+        <h4 class="ui header large">
+          <i class="fad fa-chart-line-up icon"></i>
+           Grow with us
+        </h4>
+
+        <p>
+            We have customers that range from tiny companies and projects,
+            to some of the largest in the world.
+            Read the Docs grows with you,
+            staying our of your way at a small size,
+            but has advanced authoring features when you need them.
+        </p>
+
+        <p>
+          When it's easy for your team to collaborate, your docs get updated
+          <b>more frequently</b> and with <b>less effort</b>.
+        </p>
```

Comment:
Is it possible to use `{% markdown %}` block for these to avoid having to write HTML tags?

Reply:
It is! If I found myself writing more that one paragraph, I used `{% markdown %}`